packages/locale/src/countries.mock.ts

```diff
@@ -0,0 +1,41 @@
+import { Country } from '@bigcommerce/checkout-sdk';
+
+export function getCountries(): Country[] {
```

Comment:
payment-integration-test-utils package already has this mock, no need to duplicate it, better import it from there

Reply:
This package is located in checkout-sdk project but not here.